repl/src/main/scala/ai/h2o/sparkling/repl/BaseH2OInterpreter.scala

```diff
@@ -96,8 +96,10 @@ private[repl] abstract class BaseH2OInterpreter(val sparkContext: SparkContext,
     initBeforeRunningCode(code)
     // Redirect output from console to our own stream
     scala.Console.withOut(consoleStream) {
-      try loop()
-      catch AbstractOrMissingHandler()
+      try {
+        val text = Iterator.continually(in.readLine("")).takeWhile(x => x != null).mkString("\n")
```

Comment:
Why  `""` is passed to `readline`?

Reply:
Based on the method `readLine` it seems like it's a prompt the method expects, but in this case we don't have any, so that is why ""

 ```  def readLine(prompt: String): String =
    // hack necessary for OSX jvm suspension because read calls are not restarted after SIGTSTP
    if (isMac) restartSysCalls(readOneLine(prompt), reset())
    else readOneLine(prompt)
```

Reply:
oh ok, thanks!